doc/theoretical_description_binary_classification.rst

```diff
@@ -6,41 +6,21 @@
 Theoretical Description
 =======================
 
-For binary classification in the distribution-free setting, there are
-different ways of handling uncertainty quantification: calibration,
-probabilistic prediction (consisting in estimating confidence intervals or CIs) 
-and set prediction (consisting in estimating prediction sets or PSs). Gupta et al.
-have established a tripod of theorems that connect these three notions for
-score-based classifier [1]. We will use their notation to present these different concepts.
-
-In MAPIE, we focus specifically on set prediction for multi-class classification
-and on calibration for binary classification (via scikit-learn).
-
-Although set prediction is possible for binary classification, we don't recommend using this setting.
-Instead, we recommend the use of calibration (see more details in the
-Calibration section or by using the
-:class:`~sklearn.calibration.CalibratedClassifierCV` proposed by scikit-learn
-or :class:`~mapie.calibration.MapieCalibrator` proposed in MAPIE
-even if top-Label calibration is actually not really binary use-cases).
-Here is an argument from Gupta et al [1]:
+There are mainly three different ways to handle uncertainty quantification in binary classification:
+calibration (see :doc:`theoretical_description_calibration`), confidence interval (CI) for the probability
+:math:`P(Y \vert f(X))` and prediction sets (see :doc:`theoretical_description_classification`).
+These 3 notions are tightly related for score-based classifier, as it is shown in [1]. 
 
-    PSs and CIs are only ‘informative’ if the sets or intervals produced by them are small. To quantify
-    this, we measure CIs using their width (denoted as :math:`|C(.)|)`, and PSs using their diameter (defined as
-    the width of the convex hull of the PS). For example, in the case of binary classification, the diameter
-    of a PS is :math:`1` if the prediction set is :math:`\{0,1\}`, and :math:`0` otherwise (since :math:`Y\in\{0,1\}`
-    always holds, the set :math:`\{0,1\}` is ‘uninformative’). A short CI such as :math:`[0.39, 0.41]`
-    is more informative than a wider one such as :math:`[0.3, 0.5]`.
+Prediction sets can be computed in the same way for multiclass and binary classification with
+:class:`~mapie.calibration.MapieClassifier`, and there are the same theoretical guarantees.
+Nevertheless, prediction sets are often much less informative in the binary case than in the multiclass case[#]_.
```

Comment:
What is [#]_?

Reply:
It is for footnote in the documentation.

Reply:
I think this isn't clear. Maybe another way to show this?

Reply:
I will remove it next to its call.